### PR TITLE
[hi] Localize content/en/docs/reference/glossary/addons.md to hindi

### PR DESCRIPTION
**Pull request by khannakshat7**
Signed-off-by: Akshat Khanna [khannakshat7@gmail.com](mailto:khannakshat7@gmail.com)

Localizes content/en/docs/reference/glossary/addons.md to Hindi content/hi/docs/reference/glossary/addons.md

Fixes :- #35934

### CODE REVIEW

File: content/hi/docs/reference/glossary/addons.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  संसाधन जो कुबेरनेट्स की कार्यक्षमता का विस्तार करते हैं।
+
+aka:
+tags:
+- tool
+---
+  संसाधन जो कुबेरनेट्स की कार्यक्षमता का विस्तार करते हैं।
+
+<!--more-->
+[ऐडऑन इंस्टॉल करना](/docs/concepts/cluster-administration/addons/) अपने क्लस्टर के साथ ऐड-ऑन का उपयोग करने के बारे में अधिक बताता है, और कुछ लोकप्रिय ऐड-ऑन को सूचीबद्ध करता है।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/addons.md:16]: Is it ऐड-ऑन or ऐडऑन?

```suggestion
[ऐडऑन इंस्टॉल करना](/docs/concepts/cluster-administration/addons/) अपने क्लस्टर के साथ ऐड-ऑन का उपयोग करने के बारे में अधिक बताता है, और कुछ लोकप्रिय ऐड-ऑन को सूचीबद्ध करता है।
```

**Review by divya-mohan0209 — COMMENTED** [review]
Clarification sought.

File: content/hi/docs/reference/glossary/addons.md (hunk 1 above)
Comment (khannakshat7) [reply content/hi/docs/reference/glossary/addons.md:16]: Yes I think it should be ऐड-ऑन. I'll make the change. Thank You Divya

**Comment by khannakshat7** [comment]
@mittalyashu and @anubha-v-ardhan Please review the PR

File: content/hi/docs/reference/glossary/addons.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  संसाधन जो कुबेरनेट्स की कार्यक्षमता का विस्तार करते हैं।
+
+aka:
+tags:
+- tool
+---
+  संसाधन जो कुबेरनेट्स की कार्यक्षमता का विस्तार करते हैं।
+
+<!--more-->
+[ऐड-ऑन इंस्टॉल करना](/docs/concepts/cluster-administration/addons/) अपने क्लस्टर के साथ ऐड-ऑन का उपयोग करने के बारे में अधिक बताता है, और कुछ लोकप्रिय ऐड-ऑन को सूचीबद्ध करता है।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/reference/glossary/addons.md:16]:
```suggestion
[ऐड-ऑन इंस्टॉल करना](/docs/concepts/cluster-administration/addons/) अपने क्लस्टर के साथ ऐड-ऑन का उपयोग करने के बारे में अधिक जानकारी देता है, और कुछ लोकप्रिय ऐड-ऑन को सूचीबद्ध करता है।
```
[Nit] Sounds more formal to me

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

File: content/hi/docs/reference/glossary/addons.md (hunk 2 above)
Comment (khannakshat7) [reply content/hi/docs/reference/glossary/addons.md:16]: Thank @anubha-v-ardhan for the suggestion. I have made the changes.

**Comment by khannakshat7** [comment]
@mittalyashu and @anubha-v-ardhan Please review the PR and merge it

**Review by divya-mohan0209 — COMMENTED** [review]
/lgtm

**Comment by khannakshat7** [comment]
@anubha-v-ardhan please approve the PR

/assign @anubha-v-ardhan

**Comment by amolmote** [comment]
@khannakshat7, rebase all commits into one commit, they will merge it soon.

**Comment by khannakshat7** [comment]
@amolmote I have squash the commits. Please merge it soon

**Review by anubha-v-ardhan — COMMENTED** [review]
Hey @khannakshat7, We've decided to use `main` for all our future work. Please change the base branch for this PR to `main` and we'll be ready to merge this.

**Comment by khannakshat7** [comment]
@anubha-v-ardhan I have changed the base of the PR

**Comment by divya-mohan0209** [comment]
Assuming approval from [this comment](https://github.com/kubernetes/website/pull/35961#pullrequestreview-1111489163) by @anubha-v-ardhan & providing approval.
/approve

**Pull request merged**